From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ZHANG Yuntian <yuntian@protoducer.com>
Date: Tue, 13 Sep 2022 00:07:32 +0800
Subject: [PATCH] Update build dependencies

---
 debian/control | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/control b/debian/control
index 96d01d5..f463ac7 100644
--- a/debian/control
+++ b/debian/control
@@ -6,6 +6,7 @@ Build-Depends: cmake,
                libgnutls28-dev,
                libjpeg62-turbo-dev,
                libpng-dev,
+               lintian,
                quilt,
                tar,
                unifont,
-- 
2.37.2

